third_party/amd/include/TritonAMDGPUTransforms/Passes.td

```diff
@@ -111,4 +111,14 @@ def TritonAMDGPUReorderInstructions: Pass<"tritonamdgpu-reorder-instructions", "
   let dependentDialects = [];
 }
 
+def TritonAMDGPUConvertToBufferOps : Pass<"tritonamdgpu-convert-buffer-ops", "mlir::triton::FuncOp"> {
```

Comment:
This must be called as a sub-pass or changed to operate on `ModuleOp`. In compiler.py it's called on the `ModuleOp mod`..